server/state/state_manager.js

```diff
@@ -14,116 +14,118 @@ limitations under the License.
-        console.assert(data.name in self.trackedState_);
-        self.get(data.name).set(data.value, data.time);
-      } else {
-        debug('State update for ' + data.name + ' not accepted. ' +
-            'State already owned by ' + self.stateOwners_[data.name]);
-      }
-    });
-    debug('Received late request for new state from ' + socket.id);
-    self.informLateClient_(socket);
-  });
-};
-StateManager.prototype.create = function(name, interpolatorDef, owner) {
-  console.assert(!(name in this.trackedState_));
-  if (owner === undefined) {
-    owner = 'server';
   }
-  this.trackedState_[name] = new sharedState.SharedState(
-    name, sharedState.decodeInterpolator(interpolatorDef), owner);
 
-  this.interpolatorDefs_[name] = interpolatorDef;
-  this.informClient_(name, this.network_);
-};
-// Makes an unshared SharedState instance for the private use of the server.  It
-// is not sent to clients.
-StateManager.prototype.createPrivate = function(interpolatorDef) {
-  return new sharedState.SharedState(
-    'private', sharedState.decodeInterpolator(interpolatorDef), 'server');
-};
-StateManager.prototype.createSchedule = function(interpolatorDef, schedule) {
-  return new StateSchedule(this, interpolatorDef, schedule);
-};
-StateManager.prototype.informLateClient_ = function(socket) {
-  for (var k in this.trackedState_) {
-    this.informClient_(k, socket);
+  create(name, interpolatorDef, owner) {
+    console.assert(!(name in this.trackedState_));
+    if (owner === undefined) {
+      owner = 'server';
+    }
+    this.trackedState_[name] = new sharedState.SharedState(
+      name, sharedState.decodeInterpolator(interpolatorDef), owner);
+
+    this.interpolatorDefs_[name] = interpolatorDef;
+    this.informClient_(name, this.network_);
   }
-};
-StateManager.prototype.informClient_ = function(name, socket) {
-  if (socket.id) {
-    debug('informing client ' + socket.id + ' about state', name);
-  } else {
-    debug('informing all clients about state', name);
+  // Makes an unshared SharedState instance for the private use of the server.  It
+  // is not sent to clients.
+  createPrivate(interpolatorDef) {
+    return new sharedState.SharedState(
+      'private', sharedState.decodeInterpolator(interpolatorDef), 'server');
   }
-  socket.emit('newstate', {
-    name: name,
-    interpolatorDef: this.interpolatorDefs_[name],
-    owner: this.stateOwners_[name]
-  });
-};
-StateManager.prototype.get = function(name) {
-  return this.trackedState_[name];
-};
-
-// Sends all of the most recent state to the clients.
-// This assumes that we'll call this function no more often than we change the
-// value of the variable for the latest time (which seems safe).
-StateManager.prototype.send = function() {
-  // Build a packet of data about all of the tracked state.
-  var stateData = _.map(this.trackedState_, function(state, name) {
-    return {
+  createSchedule(interpolatorDef, schedule) {
```

Comment:
Can you separate these functions with new lines in between them

Reply:
It's been fixed now